From: Markus Koschany <apo@debian.org>
Date: Wed, 3 Oct 2018 16:49:47 +0200
Subject: remove j9stub

Apparently the Java 9 stub file is not needed because we use OpenJDK8?

Fix a FTBFS due to
/build/lombok-1.16.22/src/j9stubs/org/mapstruct/ap/spi/AstModifyingAnnotationProcessor.java:21: error:
package javax.lang.model.type is not visible
[ivy:compile] import javax.lang.model.type.TypeMirror;
[ivy:compile]                        ^
[ivy:compile]   (package javax.lang.model.type is declared in module java.compiler, but module
             org.objectweb.asm does not read it)

Forwarded: no
---
 build.xml | 3 ---
 1 file changed, 3 deletions(-)

--- a/build.xml
+++ b/build.xml
@@ -273,12 +273,7 @@
 			<src path="src/core9" />
 		</javac>
 		<mkdir dir="build/lombokMapstruct" />
-		<javac includeAntRuntime="false" source="1.9" target="1.9" destdir="build/lombokMapstruct">
-			<src path="src/j9stubs" />
-			<!-- This includes org.mapstruct.ap.spi.AstModifyingAnnotationProcessor; putting this on the classpath doesn't work (needs to be internal or a module) so we just add it and then delete the class file for convenience. -->
-		</javac>
 		<mkdir dir="build/lombok/secondaryLoading.SCL.lombok/org/mapstruct/ap/spi" />
-		<move file="build/lombokMapstruct/org/mapstruct/ap/spi/AstModifyingAnnotationProcessor.class" tofile="build/lombok/secondaryLoading.SCL.lombok/org/mapstruct/ap/spi/AstModifyingAnnotationProcessor.SCL.lombok" />
 		
 		<ivy:compile destdir="build/lombok/Class50" source="1.5" target="1.6" ecj="true" nowarn="true">
 			<bootclasspath location="build/stubs" />
